src/components/textfield/TextField.js

```diff
@@ -43,7 +43,8 @@ export default class TextFieldComponent extends BaseComponent {
   elementInfo() {
     const info = super.elementInfo();
     info.type = 'input';
-
+    info.attr.id = "".concat(this.component.key);
```

Comment:
Is there a reason you are doing a concat with an empty string? If you want ensure it is a string you could use this.component.key.toString() although it should already be a string.

Reply:
No it works without the concat. 
I saw this done somewhere else in the code base and just thought there was a reason for it.